### ISSUE TITLE
Incorrect textwrap positioning with units other than "px"

### ISSUE THREAD

**Issue by miceno** [d3plus/d3plus#331]
I am integrating textwrap in a zoomable treemap visualization directly on d3.js (not using d3plus :(

The issue I have perceived is the "dy" attribute of the original text is used to position the new tspan into the "text" SVG, but the textwrap function is not considering the units of the dy attribute.

I have prepared a fiddle showing it, I have fixed "dy" to "0.75em" and textwrap uses it as it was measured in pixels. http://jsfiddle.net/miceno/jte74dj5/

**Comment by alexandersimoes** [comment]
Yeah this is strange. A hot fix if interested is to change line 16 in [this textwrap source file](https://github.com/alexandersimoes/d3plus/blob/master/src/textwrap/helpers/tspan.coffee#L16) to "em" instead of "px" and then recompile your own version of the library. OR you can [edit the compiled version](https://github.com/alexandersimoes/d3plus/blob/master/d3plus.js#L19505) manually on line 19505 and again change "px" to "em".

I'll keep this issue open and investigate how changing this effects our tests and fix it in the next d3plus release.

**Comment by miceno** [comment]
Thank you for the workaround!

I am afraid the library should take into account the kind of units developers provide, since developers will provide the values of x, y and other parameters in different units. I guess a way to "propagate" the units to the [function](https://github.com/alexandersimoes/d3plus/blob/master/src/textwrap/helpers/tspan.coffee#L16) you appointed in your comment will be great.

**Comment by miceno** [comment]
Considering it more thoroughly, the library should "normalize" units to pixels and use them accordingly, since all along the different functions, operations should be consistent. 

If "font-size" is given in "em" and "y" is given in "px", most of the operations on tspan.coffee will not be consistent.

**Comment by alexandersimoes** [comment]
True... all good points. We're working on a 2.0 branch currently and will consider these comments. Thanks for your interest in the project!
